user/pages/03.commerce2/01.getting-started/02.getting-help/docs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Getting help
+taxonomy:
+    category: docs
+---
+
+## Drupal Commerce Issue Queue
+If you are new to Drupal, you will want to learn more about [Drupal issue queues].
+
+Drupal Commerce is an actively maintained project, and oftentimes you'll be
+able to search its [Issue Queue] to find an answer to a question or a solution to
+a problem/bug. Solutions are usually presented in the form of a patch. See the
+[Patching section](../../02.developer-guide/02.install-update/07.patching) of this documentation
+guide for information about applying patches.
+
+## Chat with the Drupal Commerce community using Slack
```

Comment:
Maybe we can make it more concise: "Chat with us on Slack"

Reply:
See it in its current length: https://pr-236-4wo45ci-boljpk23hsit6.eu.platform.sh/commerce2/getting-started/getting-help